test/Bug1730993Test.php

```diff
@@ -24,28 +24,19 @@
  */
 
 use lsolesen\pel\PelJpeg;
+use PHPUnit\Framework\TestCase;
 
-class Bug1730993Test extends \PHPUnit_Framework_TestCase
+class Bug1730993Test extends TestCase
 {
-    function testThisDoesNotWorkAsExpected()
+    function testWindowWindowExceptionIsCaught()
     {
         $tmpfile = dirname(__FILE__) . '/images/bug1730993_tmp.jpg';
-        $bigfile = dirname(__FILE__) . '/images/bug1730993_large.jpg';
-        // TODO: Should not throw exception
-        $this->markTestIncomplete(
-          'This test fails and should be fixed.'
-        );
-        try {
-            require_once 'PelJpeg.php';
-            $jpeg = new PelJpeg($tmpfile); // the error occurs here
-            $exif = $jpeg->getExif();
-            if ($exif !== null) {
-                $jpeg1 = new PelJpeg($bigfile);
-                $jpeg1->setExif($exif);
-                file_put_contents($bigfile, $jpeg1->getBytes());
-            }
-        } catch (Exception $e) {
-            $this->fail('Test should not throw exception: ' . $e->getMessage());
-        }
+        $jpeg = new PelJpeg($tmpfile);
+    }
+
+    function testWindowOffsetExceptionIsCaught()
+    {
+        $tmpfile = dirname(__FILE__) . '/images/27303092-d9d72838-54f6-11e7-943c-78933f9c9fbe.jpg';
```

Comment:
Please rename that file to match it's name with the test. Could be bug1730993-2.jpg